tests/functional/query_tag/test_query_tags.py

```diff
@@ -95,20 +93,14 @@ def seeds(self):
     def project_config_update(self, prefix):
         return {
             "config-version": 2,
-            "models": {
-                "tests": {"query_tag": prefix, "post-hook": "{{ check_tag() }}"},
-            },
-            "seeds": {
-                "tests": {"query_tag": prefix, "post-hook": "{{ check_tag() }}"},
-            },
-            "snapshots": {
-                "tests": {"query_tag": prefix, "post-hook": "{{ check_tag() }}"},
-            },
+            "models": {"query_tag": prefix, "post-hook": "{{ check_query_tag() }}"},
```

Comment:
If you look at the successful runs, they warn that `models.tests`, `seeds.tests`, and `snapshots.tests` were unused configuration paths. In other words, the post-hook never ran. We also know that since the post-hook references a macro that does not exist; it should have referenced `check_query_tag` instead of `check_tag`.